src/schemas/json/hammerkit.json

```
@@ -0,0 +1,349 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "additionalProperties": false,
+  "definitions": {
+    "labels": {
+      "description": "A map of label values",
+      "type": "object",
+      "additionalProperties": { "type": "string" }
+    },
+    "envMap": {
+      "description": "A map of environment values",
+      "type": "object",
+      "additionalProperties": {
+        "type": "string"
+      }
+    },
+    "taskCommand": {
+      "anyOf": [
+        {
+          "description": "Command to be executed for the task",
+          "type": "string"
+        },
+        {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "cmd": {
+              "description": "Command to be executed for the task",
+              "type": "string"
+            },
+            "path": {
+              "description": "Path below build file path",
+              "type": "string"
+            }
+          },
+          "required": ["cmd"]
+        }
+      ]
+    },
+    "include": {
+      "type": "object",
+      "description": "A map of includes",
+      "additionalProperties": {
+        "type": "string"
+      }
+    },
```

```suggestion
    "include": {
      "description": "A map of includes for the current project",
      "type": "object",
      "additionalProperties": {
        "type": "string"
      }
    },
```

Almost the same issue as above.
Optional fix: unexpected `type` property, expected `description` before it.